flight/Modules/Stabilization/stabilization.c

```diff
@@ -219,6 +220,15 @@ static void stabilizationTask(void* parameters)
 		dT = PIOS_DELAY_DiffuS(timeval) * 1.0e-6f;
 		timeval = PIOS_DELAY_GetRaw();
 		
+		// exponential moving averaging (EMA) of dT to reduce jitter; ~200points
+		// to have more or less equivalent noise reduction to a normal N point moving averaging:  alpha = 2 / (N + 1)
+		dT_filtered = 0.01f * dT + (1.0f - 0.01f) * dT_filtered;
```

Comment:
I already noticed that all other functions in stabilization run with the unfiltered dT. 
So I am not sure if we really need filtered dT for the Gyro LPF, or if all other function should also use dT_filtered.

But as I have not much experiences with flying, changing also the PID control to this would need good testings, an I am not sure if this should be in the scope of this PR.

The implementation would be very easy, but testing might be critical.